scripts/logger_pc_app/README.md

```diff
@@ -0,0 +1,123 @@
+[INFO]  <GPIO>  GPIO Backend Initialized
+[INFO]  <RTT>   RTT Backend Initialized
+[INFO]  <UART>  Waiting for TCP connection on port 1338
+[INFO]  <GPIO>  Waiting for TCP connection on port 1337
+[INFO]  <RTT>   Waiting for TCP connection on port 1339
+>>
+```
+
+Application is up-and-running. Read next sections to know how to use backend of your interest.
+
+### Usage
+
+Application behaves similarly to console terminal. Backends are controlled using text commands, confirmed by "Enter" click. Each command has to be preceded
+with backend name, e.g. to change Saleae acquisition time to two seconds:
+
+```
+>> gpio time 2.0
+```
+
+Moreover, it is possible to control application using TCP terminal, e.g. PuTTY. Connection port can be provided during initialization or in config file.
+There is different TCP socket available for each backends, so all I/O done using this method is connected directly to specific backend process.
+It means it can be used to filter target device output by backends. Note that commands orderer over TCP does not have to be preceded by backend name
+(as they are already connected to relevant backend).
+
+Must-have PuTTY settings:
+ - Session:
+    - Connection type:                     Raw
+    - Port:                                Same as set in config file or during startup
+ - Terminal:
+    - Local echo:                          Force off
+    - Local line edit:                     Force off
+    - Keyboard:
+        - The function keys and keypad:     Linux
+
+List of available commands is printed after typing "help" command.
+
+#### GPIO backend
+
+In order to use GPIO parallel backend, Saleae Logic Analyzer has to be used. It is recommended to use Saleae Pro, due to higher sampling rates.
+
+9 GPIO pins are needed - 8 for data and 1 for clock signal. These are selected using KConfig file and can be changed by typing "ninja menuconfig"
+in target device source location. Pins should be connected in ascending order - INPUT1 to LSB, ... , INPUT8 to MSB and
+clock signal to INPUT9, where INPUTx are consecutive Saleae inputs. Do not forget about connecting Saleae GROUND pin to target device ground.
+
+If not already done, download and install Saleae Logic software from offical site. Launch it, enter "Options" (in right upper corner) and then "Preferences".
+Under "Developer" tab tick "Enable scripting socket server", with default listening port.
+
+Launch ZephyrLogParser:
+
+```
+python3 zephyrlogparser.py --elf [ELF_FILE_LOCATION] --gpio-on
+```
+
+Type "init" command to connect to Saleae Logic software, and then "start" to launch data acquisition. Type "parse" to start parsing data dump.
+
+Remember about target device reset after starting data collection, due to fact that synchronization frame is used to prevent log corruption and it is send
+during logger system initialization.
+
+#### UART backend
+
+In order to use UART backend connect target device with PC host via USB and launch application:
```

Comment:
if this it this compressed uart backed then it should also have separate documentation.

Reply:
This is only the script part, UART backend will be part of a separate PR.